.gitignore

```diff
@@ -3,8 +3,8 @@ logs
 *.log
 npm-debug.log*
 yarn-debug.log*
-yarn-error.log*
-pnpm-debug.log*
```

Comment:
Why did you change this 2 lines? I think you changed these by mistake. Right?

Reply:
yep you are right